rapid/client/executor.py

```diff
@@ -92,15 +94,39 @@ def verify_work_request(self):
             raise Exception("No command was set.")
 
     def _create_logger(self):
-        logger = logging.getLogger("rapid")
+        log_file_name = self._get_log_file_name()
+        if not log_file_name:
+            logger = logging.getLogger('rapid')
+            logger.setLevel(logging.INFO)
+            return logger
+
+        logger = logging.getLogger("rapid-executor")
         logger.setLevel(logging.INFO)
+
+        self._file_handler = logging.FileHandler(log_file_name)
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        self._file_handler.setFormatter(formatter)
+        logger.addHandler(self._file_handler)
+        
         return logger
 
+    def _get_log_file_name(self):
+        if self.rapid_config and self.rapid_config.workspace and self.rapid_config.log_to_directory:
+            try:
+                os.makedirs(os.path.join(self.rapid_config.log_to_directory), exist_ok=True)
+            except:  # pylint: disable=bare-except
+                pass
+            return os.path.join(self.rapid_config.log_to_directory, '{}.log'.format(self.work_request.action_instance_id))
+        return None
+
     def start(self, threaded=True):
         """
         Spin off a process that will execute and run the given work request.
         :return:
         """
+        if self._get_log_file_name():
+            rapid_logger.info("Running: {}".format(self._get_log_file_name()))
```

Comment:
in case someone is at the wheel, we tell them the log file we're going to output everything to. Seems kind. magnanimous, even.

Reply:
It allows us track what is running through the main log and when it finishes.